sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/DotNetSensorTest.java

```diff
@@ -124,7 +124,6 @@ public void whenNoProtobufFiles_shouldNotFail() {
     assertThat(logTester.logs(LoggerLevel.DEBUG)).isEmpty();
     assertThat(logTester.logs(LoggerLevel.INFO)).containsExactly("TEST PROJECTS SUMMARY");
     assertThat(logTester.logs(LoggerLevel.WARN)).containsExactly("No protobuf reports found. The " + SHORT_LANG_NAME + " files will not have highlighting and metrics. You can get help on the community forum: https://community.sonarsource.com");
-    verify(analysisWarnings, never()).addUnique(any());
```

Comment:
should bring back? here and everywhere `verify(analysisWarnings, never()).addUnique(any());` was used